.github/workflows/cloud_tests.yml

```diff
@@ -0,0 +1,91 @@
+name: Cloud Tests
+on:
+  push:
+    branches:
+      - 'master'
+  pull_request:
+  workflow_dispatch:
+
+env:
+  PICARD_TEST_INPUTS: gs://hellbender/test/resources/
+  PICARD_TEST_STAGING: gs://hellbender-test-logs/staging/
+  PICARD_TEST_LOGS: /hellbender-test-logs/build_reports/
```

Comment:
Should we have separate locations for picard ?

Reply:
We can change them if we want.  Since we're using the same service account and set of permissions I figured it was easiest to put them in the same place where permissions and bucket lifecycles are already configured.  They shouldn't conflict with each other in any way that I can think of.